jekyll/_cci2/contexts.md

```diff
@@ -21,13 +21,15 @@ Contexts are created on the Settings page of the CircleCI application, in the Or
 
 To use environment variables set on the Contexts page, the person running the workflow must be a member of the organization for which the context is set and the rule must allow access to all projects in the org. The default context name is `org-global` and the default rule allows access to all projects in the org.
 
+Context names must be unique for each Github or Bitbucket organization. For CircleCI installations on your own server or private cloud, Context names must be unique across the installation.
+
 ## Using a Context
 
-1. Navigate to the Settings > Contexts page in the CircleCI application. The Create a Context button appears.
+1. Navigate to the Settings > Contexts page in the CircleCI application. The Create a Context button appears if you have no existing contexts.
```

Comment:
I think in the post multi contexts world, the `create context` button _always_ exists

Reply:
It just depends on where on the screen it is (ie, in the empty state or in the header)

Reply:
oh right, thanks, I removed that sentence.